packages/ui-library/src/components/position-progress-bar/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { LinearProgress, LinearProgressProps } from '@mui/material';
+import { baseColors, colors } from '../../theme';
+import { withStyles } from 'tss-react/mui';
+
+export const PositionProgressBar = withStyles(
+  ({ variant = 'determinate', ...props }: LinearProgressProps) => <LinearProgress variant={variant} {...props} />,
+  ({ palette: { mode } }, { value }) => ({
+    bar: {
+      background:
+        value === 100
+          ? colors[mode].semantic.success.darker
+          : mode === 'light'
+          ? `linear-gradient(270deg, ${colors.light.violet.violet500} 0%, ${colors.light.violet.violet400} ${value}%)`
```

Comment:
we could probably define a variable for this on the theme like "linear progress"

Reply:
At first I did! But I noticed that the last % value is dynamic. We could do a function but I don't think it's going to be included as a theme color